spec/lib/mercury/cps_spec.rb

```diff
@@ -10,40 +10,40 @@
 
   describe '::lift' do
     it 'CPS-transforms a non-CPS proc' do
-      expect(Cps.lift{rand}.run).to be_a Numeric
+      expect(Cps.lift { rand }.run).to be_a Numeric
     end
     it 'raises an error if the block returns a Cps object' do
-      expect{lift{lift{'cps'}}.run}.to raise_error /returned a Cps/
+      expect { lift { lift { 'cps' } }.run }.to raise_error /returned a Cps/
```

Comment:
Yeah... the spaces don't make this line any more readable to me. They're just distracting.

I'd go so far as to say that the only time I _want_ to see spaces inside of curly braces is when a variable is being captured, such as `foo.map{ |x| ... }`